docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -464,6 +464,34 @@ safer to use the array notation for variables.
 
 Also array notation allows for dynamic variable composition, see dynamic_variables_.
 
+
+.. _argsplat_unsafe:
+
+When is it unsafe to bulk-set task arguments from a variable?
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+
+Ansible allows all of a task's arguments to be bulk-set from a dictionary-typed variable, but issues a warning that the practice is unsafe. For example::
+
+    #...
+    vars:
+      usermod_args:
+        name: testuser
+        state: present
+        update_password: always
+    tasks:
+    - user: '{{ usermod_args }}'
+
+This technique can be useful in some dynamic execution scenarios. However, it is important to define the argument
+dictionary variable at a relatively high level of precedence, specifically, anything greater than *host facts* in the
```

Comment:
Wonder if "at a relatively high level of precedence, specifically, anything" can be removed and wording tweaked a bit